docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -33,83 +36,185 @@ Instead of sampling from a univariate distribution for each risk exposure at ini
+  Add a generalized version of this example showing that the multiplicative approach is always an overestimate of PAF.
 
-Risks with effects on the same outcome
---------------------------------------
+Proof that biases do not cancel out
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-When we introduce correlation between two risk factors that have effects on the same outcome, we have to
-be especially careful about double-counting issues.
+Since the bias discussed in the previous section causes an overestimate of the joint PAF,
+and the :ref:`bias in the GBD calculation method for individual PAFs in the presence of confounders <pafs>`
+causes an underestimate of each PAF (for positively-correlated, RR > 1 risks),
+it is reasonable to wonder if these factors cancel each other out and lead to accurate PAFs.
 
-The population-attributable fractions (PAFs) must be calculated correctly,
-or the population-level incidence of the outcome will
-not validate to GBD.
-**The usual Vivarium strategy to calculate joint PAFs (PAFs of more than one risk factor on an outcome)
-is biased when the risk factors are correlated.**
+They do not, as the bias in the calculation of the individual PAFs is always greater,
+leading to an overall underestimate.
 
-The true population attributable fraction for the combined effect of two risk factors,
-:math:`r1` and :math:`r2`, on an outcome :math:`O` such as a mortality rate is (assuming no effect modification, as we typically do):
+If we assume that there is no effect modification/there are no interaction effects,
+as we typically do, **and** we assume that there are no *additional* confounders besides
+r1 and r2,
+the true population attributable fraction for the combined effect of two risk factors,
+:math:`r1` and :math:`r2`, on an outcome :math:`O` such as a mortality rate is:
 
 .. math::
   O = O_{tmrel} * RR_{r1} * RR_{r2}
 
 .. math::
-  PAF = \frac{E(O) - E(O_{tmrel})}{E(O)} = 1 - \frac{O_{tmrel}}{E(O_{tmrel} * RR_{r1} * RR_{r2})} = 1 - \frac{1}{E(RR_{r1} * RR_{r2})}
-
-where :math:`O_{tmrel}` is a constant representing the outcome (rate) among those at the
-theoretical minimum risk exposure level.
+  PAF_{true} = \frac{E(O) - E(O_{tmrel})}{E(O)} = 1 - \frac{O_{tmrel}}{E(O_{tmrel} * RR_{r1} * RR_{r2})} = 1 - \frac{1}{E(RR_{r1} * RR_{r2})}
 
-Default approach
-^^^^^^^^^^^^^^^^
+where :math:`O_{tmrel}` is a constant representing the outcome (e.g. incidence rate) among those at the
+theoretical minimum risk exposure level for both risk factors.
 
-By default in Vivarium, a joint PAF is calculated with the "multiplicative" approach:
+In the default configuration of Vivarium, using GBD PAFs with the multiplicative approach:
 
 .. math::
-	PAF_{multiplicative} = 1 - (1 - PAF_{r1}) \cdot (1 - PAF_{r2}) = 1 - \frac{1}{E(RR_{r1})} \cdot \frac{1}{E(RR_{r2})}
+	PAF_{vivarium} = 1 - (1 - PAF_{r1}) * (1 - PAF_{r2}) = 1 - \frac{1}{E(RR_{r1})} * \frac{1}{E(RR_{r2})}
```

Comment:
Just noting that we can only assume that E(O) = O_tmrel * E(RR_r1 * RR_r2) because we've stated that there are no other confounders between r1 and r2 and O, but in most realistic situations this would not be the case because of the confounding bias. I am not sure how to approach this mathematically in a simple way without running involved nanosim tests, but I expect that this could complicate the conclusion that the underestimation is always greater than the overestimation - perhaps this would be worth discussing in a doc session or at least noting in the docs.

Reply:
I think we discussed this in the docs session, but just to leave a paper trail:

This is far from our only problem in the (extremely common) case where we leave known confounders out of a model because there are too many to simulate. In that case, the conclusions we make about the impact of intervening on a risk factor are all biased by not capturing the non-causal portion of the correlation between risk factor and outcome.

Long-term, the solution to this may be something along the lines of simulating a correlated risk "X-factor". If that is the solution, we would be moving more toward *making* the assumptions of this section true, rather than handling when they are not.